### PR TITLE
Side menu and its items are now tabbable

### PR DESCRIPTION
**Pull request by rkomiyama**
## Description of the PR
Allow side menu to be tabble, along with its items.
tabbing around

### Issues Related
#261

### CODE REVIEW

File: app/renderer/components/navbar_menu.js [hunk 1]
```diff
@@ -10,12 +10,12 @@ module.exports = {
     template: `
     <ul id='navbar-menu' class='dropdown-content'>
         <template v-if="!editMode">
-            <li class="unselectable-text menu-button"><a v-on:click="selected($event,'run-suite')">Run Suite</a></li>
-            <li class="unselectable-text menu-button"><a v-on:click="selected($event,'stop-suite')">Stop Suite</a></li>
+            <li class="unselectable-text menu-button"><a v-on:click="selected($event,'run-suite')" v-bind:href="'#tab0'">Run Suite</a></li>
```
Comment (angrykoala) [inline app/renderer/components/navbar_menu.js]: Do we really need `v-bind:href="'#tab`? I think there are easier native ways to achieve tabbing

File: test/front-end.test.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+const Application = require('spectron').Application;
+const assert = require('assert');
+
+describe('Front-end tests', function () {
+    this.timeout(20000); // needed for Travis builds
+
+    beforeEach(() => {
+        this.app = new Application({
```
Comment (angrykoala) [inline test/front-end.test.js]: I think using `this` in a test case is a bit obscure, Could you please create a `const` variable `app` in the describe and use that?

File: test/front-end.test.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+const Application = require('spectron').Application;
+const assert = require('assert');
+
+describe('Front-end tests', function () {
```
Comment (angrykoala) [inline test/front-end.test.js]: Please, use arrow function here

File: test/front-end.test.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+            })
+            .then((element) => {
+                return this.app.client.elementIdCssProperty(element.value.ELEMENT, 'visibility');
+            })
+            .then((visibility) => {
+                assert.equal(visibility.value, "visible");
+            });
+    });
+
+    it('Allows selection of side menu and its items with "Tab" key', () => {
```
Comment (angrykoala) [inline test/front-end.test.js]: Nice making this test! thanks

File: test/front-end.test.js (hunk 3 above)
Comment (rkomiyama) [reply test/front-end.test.js]: Normally I would, but I won't be able to use `this.timeout()` if I use the arrow function here.

File: test/front-end.test.js (hunk 2 above)
Comment (rkomiyama) [reply test/front-end.test.js]: Could you show me an example of what that would look like?

File: app/renderer/components/navbar_menu.js (hunk 1 above)
Comment (rkomiyama) [reply app/renderer/components/navbar_menu.js]: A simple `href` might work.

File: test/front-end.test.js (hunk 3 above)
Comment (angrykoala) [reply test/front-end.test.js]: You are right, we should use `function()` on the `describe` block on the other tests

File: app/renderer/components/navbar_menu.js (hunk 1 above)
Comment (angrykoala) [reply app/renderer/components/navbar_menu.js]: I'm not sure you even need href, I think there is a tabable syntax or something like this

File: test/front-end.test.js (hunk 2 above)
Comment (angrykoala) [reply test/front-end.test.js]:
```
describe("...",function(){
let app; //Sorry, it should be let, not const
beforeEach(()=>{
    app=new Application
    ....
})
```

This way, you explicitly know what variables are using, and the internals of the suite this won't collide

**Pull request merged**